{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/utils/external-links/index.js

```diff
@@ -0,0 +1,63 @@
+/*
+External links (defined as all links pointing to a domain that is not the
+same as the one from which the current document is being served)
+will have their 'target' attribute set to '_blank'.
+
+Social media sharing links will show in a popup.
+*/
+export default function () {
+  const links = [].slice.call(document.getElementsByTagName('a'))
+
+  // If the link starts with any of these things, we'll open a 600x300 popup
+  // window for them.
+  const usePopupPrefixes = [
+    'https://twitter.com/intent/tweet?',
+    'https://www.facebook.com/sharer.php?',
+    'https://www.linkedin.com/shareArticle?'
+  ]
+
+  let thisDomain = window.location.hostname
+
+  if (thisDomain.indexOf('www.') === 0) {
+    thisDomain = thisDomain.substr(4)
+  }
+
+  for (const link of links) {
+    const href = link.getAttribute('href')
+
+    if (!href) {
+      continue
+    }
+
+    if (href.indexOf('http://') === 0 || href.indexOf('https://') === 0 || href.indexOf('//') === 0) {
+      let domain = link.hostname
+
+      if (domain.indexOf('www.') === 0) {
+        domain = domain.substr(4)
+      }
+
+      if (domain !== thisDomain) {
+        link.setAttribute('target', '_blank')
+        // Add noopener and 'noreferrer' to work around this:
+        // https://dev.to/ben/the-targetblank-vulnerability-by-example
+        let rel = link.getAttribute('rel')
+        if (!rel) {
+          // will be null if it is not set
+          rel = ''
+        }
+        link.setAttribute('rel', `${rel} noopener noreferrer`)
```

Comment:
I always tend to have a white space before and after `if () {}` `for () {}` just for legibility.